src/jarabe/frame/activitiestray.py

```diff
@@ -411,7 +411,7 @@ def __init__(self, file_transfer):
         ToolButton.__init__(self)
 
         self.file_transfer = file_transfer
-        file_transfer.connect('notify::state', self.__notify_state_cb)
+        self.file_transfer.connect('notify::state', self.__notify_state_cb)
```

Comment:
This is not correct.  (a) the warning is wrong, as IncomingFileTransfer is a subclass of BaseFileTransfer, which is a subclass of GObject.GObject, which certainly does have _connect_, and (b) the way you change it makes no sense; both variables will point to the same object instance.

Reply:
Yeah I agree now